### ISSUE TITLE
Unity 5.4

### ISSUE THREAD

**Issue by tetTris11** [RaduMC/UnityStudio#8]
although Unity 5.4 is still in beta there are already some stuff that uses it, as such would you be able to update Unity Studio to 5.4? at the moment it doesn't  extract textures (iv only had luck with 1 texture) and does a very small number meshes. if you want i can send over some files to help

**Comment by Mactastic1-5** [comment]
No, the owner of this repository is awol. Use a different tool: [Google](http://lmgtfy.com/?q=unity+extract)
